components/elm/src/main/elm_varcon.F90

```diff
@@ -248,7 +248,7 @@ subroutine elm_varcon_init()
     allocate( dzsoifl(1:nlevsoifl            ))
 
     if (use_extrasnowlayers) then
-       h2osno_max = 10000._r8
+       h2osno_max = 30000._r8
```

Comment:
I don't think we want to be changing this in this PR.  @stephenprice , on the call today, Chloe described a new PR she will make that includes this plus a few firn parameter tuning improvements that were used in the long firn spinups.

Reply:
That's fine, but a few weeks ago she asked me to just include this here. I'm happy to remove it.

Reply:
@chloewhicker recently opened a PR that includes this change with a few other changes for the firn model in https://github.com/E3SM-Project/E3SM/pull/6298, so I think it should be removed from this PR.

Reply:
Sorry @stephenprice I didn't know I needed to submit the firn model changes when we talked about it. Matt is right, this line is included in my PR here: https://github.com/E3SM-Project/E3SM/pull/6298#pullrequestreview-1958523608

Reply:
Ok. Not sure I need to revert this commit though as it will resolve itself in the merge, right?

Reply:
#6298 has been merged already, so I think it is safe to leave this change here.  I looked a the git history, and this change was part of a commit with changes we want to keep, so dropping this change is not as simple as just removing a commit from the branch history.  I'm ok with leaving it in, unless @jonbob thinks it would be cleaner to remove it from the history or change it back.

Reply:
If it only affects cases with an active ice sheet model, its fine.